src/kite.js

```diff
@@ -57,6 +57,20 @@ const Kite = {
 
     Logger.LEVEL = Logger.LEVELS[vscode.workspace.getConfiguration('kite').loggingLevel.toUpperCase()];
 
+    KiteAPI.isKiteInstalled().catch(err => {
+      if (err.message.includes("Unable to find Kite application install")) {
+        vscode.window.showInformationMessage('Unable to find Kite application install.', 'Install').then(item => {
```

Comment:
language nit: "application" => "engine" I think adds more clarity for a user about the nature of the situation

Reply:
I wonder if adding a second sentence "The Kite engine is needed to power the completions of the Kite extention" is worthwhile too?

Reply:
Yeah, I like that. I cleaned up the language a little bit, here's the latest revision.

<img width="1136" alt="Screen Shot 2019-04-26 at 10 54 21 AM" src="https://user-images.githubusercontent.com/47993402/56826865-1c990080-6812-11e9-9a01-8652b948f852.png">